picasso/src/main/java/com/squareup/picasso/Downloader.java

```diff
@@ -46,9 +45,9 @@
     final boolean localCacheOnly;
     final int responseCode;
 
-    public ResponseException(String message, boolean localCacheOnly, int responseCode) {
+    public ResponseException(String message, int networkPolicy, int responseCode) {
```

Comment:
Sorry for late comment, but I actually have two questions:
1. Why it's exceptional? It seems changing existing public method signature.
2. Why it's _networkPolicy_ is _int_ while _NetworkPolicy_ is defined as _enum_ ?